assets/js/blocks/cart-checkout/cart/attributes.js

```diff
@@ -24,6 +25,10 @@ const blockAttributes = {
 		type: 'number',
 		default: 0,
 	},
+	hasDarkControls: {
+		type: 'boolean',
+		default: HAS_DARK_EDITOR_STYLE_SUPPORT,
```

Comment:
> If a theme is predominantly "dark", it's likely to want to opt-in to our dark mode for cart/checkout elements.
Themes that have user-defined color options (e.g. Storefront) won't get the benefit of the default, because they won't declare "dark"ness, since it's customizer dependent.

Yes, we have this method to assume a theme is dark:
- The themes supports `dark-editor-style`, by explicitly supporting this, most editor borders and placeholders will be white, making them illegible on white background, by this we can assume the background is dark.

A theme can also register support for `custom-background`, however I'm not aware if there's a standardized way of fetching that value, `get_theme_support` will only fetch the default value.
If we can fetch the custom-background value, we can run some calculations (like TwentyTwenty) to see if the background is dark or light, and set the default based on that.